server/chat-plugins/scavengers.ts

```diff
@@ -1483,9 +1486,16 @@ const ScavengerCommands: ChatCommands = {
 		this.checkCan('mute', null, room);
 		const game = room.getGame(ScavengerHunt);
 		if (!game) return this.errorReply(`There is no scavenger hunt currently running.`);
+		
+		let minutes;
+		if (target == "off") {
+			minutes = 0;
+		}
+		else {
+			minutes = parseFloat(target);
+		}
```

Comment:
```suggestion

		const minutes = (toID(target) === 'off' ? 0 : parseFloat(target));
```

Reply:
(To be clear, `game.setTimer(0)` should clear the timeout, not set it to 0 milliseconds, right?)

Reply:
That is correct

Reply:
Also, if you were to do this with an if statement, I'd do it like this:
```ts
let minutes = 0;
if (target !== 'off) minutes = parseFloat(target);
```
or
```ts
let minutes;
if (target === 'off') {
     minutes = 0;
} else {
     minutes = parseFloat(target);
}
```

Reply:
Thanks. I will remember this in the future. For now, I've just used the 1-line const declaration